lib/rataextra-backend.ts

```diff
@@ -74,8 +104,22 @@ export class RataExtraBackendStack extends NestedStack {
       relativePath: '../packages/server/lambdas/dummy2.ts',
     });
 
+    const createUser = this.createNodejsLambda({
+      ...prismaParameters,
+      name: 'create-user',
+      relativePath: '../packages/server/lambdas/create-user.ts',
+    });
+
+    const listUsers = this.createNodejsLambda({
+      ...prismaParameters,
+      name: 'list-users',
+      relativePath: '../packages/server/lambdas/list-users.ts',
+    });
+
     // Add all lambdas here to add as alb targets
     const lambdas: ListenerTargetLambdas[] = [
+      { lambda: listUsers, priority: 70, path: ['/api/users'] },
```

Comment:
@mehtis is it a good idea we number every lambda request's priority manually ourselves? Could a duplicate or wrong priority number happen if the request number grows later? (just curious :octocat: )

Reply:
I'd expect the synth to fail, if there were duplicate numbers, so that wouldn't go through. My best guess, though. It is possible that a priority number could be in the wrong order. But this is generally an issue only with wildcard values. If we generally use specific paths, then the order doesn't really matter. Maybe a comment could be added here noting to keep the list ordered by priority and to be extra careful with wildcard paths @saneDG

Reply:
Added comment to guide priority usage